### ISSUE TITLE
Merch explorer "sitc" and "Facet on" fieldset tag missing

### ISSUE THREAD

**Issue by mrjoh3** [djpr-data/djprtradedash#106]
this appears to be an issue with the R package that produces the radio buttons. I think it could be fixed by adding `role = "fieldset"` to the div. Hopefully this can be done in the R function, if not ask @mrjoh3 how to us JS to add the role option
